pkg/loopback/loopback_test.go

```diff
@@ -0,0 +1,136 @@
+// +build linux,cgo
+
+package loopback
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"strings"
+	"syscall"
+	"testing"
+
+	"fmt"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const filePath = "/tmp/"
+
+var rootEnabled bool
+
+func init() {
+	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
@dnephin The problem is running a destructive test as `uid=0` and not expecting to have the test run whatever it did. This makes root-level tests opt-in, rather than opt out.

Let's not assume that everyone is using moby's build system to build everything and that it is widely safe. As it is, they take way to long to run, so assuming that this will always be safe to run as uid 0 is a poor assumption.